pkg/maps/ctmap/gc/gc.go

```diff
@@ -21,12 +21,55 @@ import (
 
 	"github.com/cilium/cilium/pkg/bpf"
 	"github.com/cilium/cilium/pkg/endpoint"
+	"github.com/cilium/cilium/pkg/endpointmanager"
+	"github.com/cilium/cilium/pkg/logging"
 	"github.com/cilium/cilium/pkg/logging/logfields"
 	"github.com/cilium/cilium/pkg/maps/ctmap"
-
 	"github.com/sirupsen/logrus"
 )
 
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "ct-gc")
+
+// EnableConntrackGC enables the connection tracking garbage collection.
+func EnableConntrackGC(ipv4, ipv6 bool, restoredEndpoints []*endpoint.Endpoint, mgr *endpointmanager.EndpointManager) {
```

Comment:
Don't import the endpointmanager directly, use an interface instead to avoid cycles in the future. (By extension we could almost do the same for the `endpoint.Endpoint`. Feel free to open a GH issue good-first-issue for this last one)

Reply:
Will add an interface for `EndpointManager`. The last one is a bit more difficult because `GetEndpoints` returns a slice of Endpoint pointers; we can't have the return type for `GetEndpoints` not be a slice of `*endpoint.Endpoint`, because that means that `GetEndpoints` won't actually implement that interface :/ .

Reply:
Added the interface per your request, PTAL